src/components/CriteriaList/ImportCriteriaModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, useCallback, useRef, memo, FormEvent } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFileImport, faTimes } from '@fortawesome/free-solid-svg-icons';
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Input,
+  IconButton
+} from '@material-ui/core';
+
+import { useCriteriaContext } from 'components/CriteriaProvider';
+import useStyles from './styles';
+
+interface ImportCriteriaModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const ImportCriteriaModal: FC<ImportCriteriaModalProps> = ({ open, onClose }) => {
+  const styles = useStyles();
+  const importFileRef = useRef<HTMLInputElement>(null);
+
+  const { addCriteria } = useCriteriaContext();
+
+  const selectFile = useCallback(
+    (event: FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+      const files = importFileRef.current?.files;
```

Comment:
it looks like `importedFileRef` defaults to null, so it might be prudent to have an additional optional chaining operator here

```suggestion
      const files = importFileRef?.current?.files;
```